app/views/episodes/_form_media.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="prx-uploads col-12 mb-4">
+  <div class="card shadow border-0">
+
+    <div class="card-header card-header-info d-flex justify-content-between">
+      <h5 class="card-title"><%= t(".title") %></h5>
+    </div>
+
+    <div class="card-body">
+      <%= turbo_frame_tag "episode-form-media" do %>
+        <div class="row">
+
+          <div class="col-12 mb-4 prx-field-group">
+            <div class="form-floating" data-controller="dynamic-form">
+              <%= form.select :medium, episode_medium_options, {}, data: {action: "dynamic-form#change"} %>
```

Comment:
"Medium" is a new field.  It's in a turbo frame _outside_ of the segment count.  Since we don't display the segment count for videos.